worlds/mmbn3/Names/LocationName.py

```diff
@@ -0,0 +1,275 @@
+## Blue Mystery Datas
+# ACDC Area
+ACDC_1_Southwest_BMD = "ACDC 1 Southwest BMD"
```

Comment:
Nit: Instead of writing these out like this, adding them to a dictionary (or maybe a few different dictionaries) would take up a similar amount of space in this file, and allow you to enumerate over them much faster in other files (instead of having to write them out in multiple places).

Reply:
The idea of this is to have the names be compile-time constants to avoid typos causing issues. I'm not sure how a dictionary would solve that issue, as I'd still need compile-time constants for each key in the dictionary mapping to a location name.

Reply:
containing in a dictionary/map just allows you to iterate over some of the items (which is done in a few spots like Regions.py), while also retaining the ability to search for individual variable names. For example you could do something like : 
###PseudoCode

all acdc regions = {
regionVar1 = "regionName1", 
regionVar2 = "regionName2"
etc
}

all scilab regions = {...}
etc

allRegions.addAll(acdcRegions)
allRegions.addAll(scilabRegions)

Then you can iterate over the scilab areas for certain things that need iteration (like Regions.py), and then still access vars in allRegions.

But, as I prefaced my original comment with, this is just me being nitpicky more than anything (I think it'd cut down ~250-300 lines of code).